internal/buildscripts/packaging/msi/splunk-support-bundle.ps1

```diff
@@ -0,0 +1,263 @@
+# Copyright Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+#######################################
+# Globals
+#######################################
+$CONFDIR="C:\Program Files\Splunk\OpenTelemetry Collector" # Default configuration directory
+$DIRECTORY= # Either passed as CLI parameter or later set to CONFDIR
+$TMPDIR="C:\Program Files\Splunk\splunk-support-bundle-$([int64](New-TimeSpan -Start (Get-Date "01/01/1970") -End (Get-Date)).TotalSeconds)" # Unique temporary directory for support bundle contents
```

Comment:
```suggestion
$TMPDIR="${env:PROGRAMFILES}\Splunk\OpenTelemetry Collector\splunk-support-bundle-$([int64](New-TimeSpan -Start (Get-Date "01/01/1970") -End (Get-Date)).TotalSeconds)" # Unique temporary directory for support bundle contents
```

Reply:
Hi @jcheng-splunk 
It is better to keep TEMPDIR outside  `OpenTelemetry Collector` directory because while [copying configuration](https://github.com/djani-splunk/splunk-otel-collector/blob/main/internal/buildscripts/packaging/msi/splunk-support-bundle.ps1#L104) it will copy the same directory and goes in an infinite loop.